scRNA-seq/03-dimension_reduction_scRNA-seq.Rmd

```diff
@@ -20,63 +20,62 @@ downstream analyses.
 
 ## Set Up 
 
-For these analyses, we will need Seurat as well as the dplyr pipe. 
-
 ```{r}
-library(Seurat)
+# Load libraries
+library(ggplot2)
 
 # Magrittr pipe
 `%>%` <- dplyr::`%>%`
 ```
 
-We need to reload the Seurat object with the data we set up and normalized 
-previously. 
-
-```{r Path to Seurat}
-# Load in Seurat object
-seurat_norm <- readRDS(file.path("data", "seurat_object.RDS"))
+For this module, we are going to use the dataset that we processed part of in 
+`02-tag-based_pre-processing_scRNA-seq.md`
+We have already filtered and `scran` normalized this data for you (following 
+the same kinds of steps we did in `01-normalizing_scRNA-seq.Rmd`).
+We also randomly subsampled 500 cells, and are only using the top 95% highest 
```

Comment:
You have filtered out 5% or you're only using 5%? I think it's the first one based on the sentence but can see how this could be ambiguous.

Reply:
Can you tell me a little bit about how you picked this cutoff?

Reply:
Filtered out bottom 5%. I picked it fairly arbitrarily but then when I did PCA and tissue labels it seemed still representative of the data as a whole so I kept it.

Reply:
The 500 samples bit I came to by reducing the amount of samples a little bit each time and running the script to see how long it took. Was looking for something that took less than ~10 seconds.

Reply:
Did you randomly subsample the reads from 500 cells? I think the answer is no but could be ambiguous.

Reply:
No, just randomly chose columns of data.